doc/releases/release-notes-3.3.rst

```diff
@@ -510,6 +510,7 @@ Libraries / Subsystems
   * Added the option to disable CRC checking in :ref:`fcb_api` by enabling the
     Kconfig option :kconfig:option:`CONFIG_FCB_ALLOW_FIXED_ENDMARKER`
     and setting the `FCB_FLAGS_CRC_DISABLED` flag in the :c:struct:`fcb` struct.
+  * Added :kconfig:option:`CONFIG_FS_FATFS_REENTRANT` to enable the FAT FS reentrant option.
```

Comment:
This will need to move to 3.4, but do that after 3.3 release is finished as the file will be created then

Reply:
Makes sense. However, do you know when 3.3 will be released and the 3.4 file created? - Doesn't this hold up all pending PRs?